fil_fungible_token/src/token/mod.rs

```diff
@@ -623,6 +614,32 @@ where
         }
     }
 
+    /// Calls the receiver hook, do not revert on error
+    /// Error results from this should be handled upstream
```

Comment:
I think "do not revert on error" doesn't make much sense to specify here. We can just say it calls the receiver hook, and returns an error if it fails, in response to which most callers should abort.